### PR TITLE
added Themes & media Queries

### PR DESCRIPTION
**Pull request by WE-dev1**
also fixed some bugs and polished the design.

### CODE REVIEW

File: utils/tileClickTest.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      setShowWorkout,
+      setShowHistoryEntry,
+      setHistoryEntryData,
+      showHistoryEntry,
+      isWorkoutToday,
+      showWorkout
+    )
+  ).toThrow();
+});
+ */
```
Comment (AnjaSiewert) [inline utils/tileClickTest.js:190]: Do you still need this code? Otherwise you can delete it

**Review by AnjaSiewert — COMMENTED** [review]
LGTM 🚀 Your app looks great! :)
You used a lot of inline styling, I did not comment on all of them. StyledComponent would be a better option, especially when you are using several styling props on one tag.

File: components/Exercise/index.js [hunk 2]
```diff
@@ -154,6 +156,7 @@ export default function Exercise({
       >
         <AddExerciseForm>
           <Input
+            style={{ border: "1.5px solid var(--2)" }}
```
[the remark this answers is not in the document]
Comment (WE-dev1) [reply components/Exercise/index.js:159]: You are right. In some cases I think I use the component multiple times though. But no time to clean up the whole css anymore😅

File: utils/tileClickTest.js (hunk 1 above)
Comment (WE-dev1) [reply utils/tileClickTest.js]: Thank you for adding up to my cleanup todos😊

**Pull request merged**